### PR TITLE
Updates two known issues and corrects FAQ

### PR DESCRIPTION
**Pull request by aturner-epcc**
Adds notes on libfabric memory leak issue and fact that FFTW points to Haswell version at runtime.

### CODE REVIEW

File: docs/known-issues/index.md [hunk 1]
```diff
@@ -6,6 +6,40 @@ active investigation by HPE Cray and the wider service.
...
+module use the correct version of the libraries at runtime, you should add the following lines to your job submission
+script:
+
+```
+module load cray-fftw
+export LD_LIBRARY_PATH=$CRAY_LD_LIBRARY_PATH:$LD_LIBRARY_PATH
+```
+
+This will make sure that the application uses the correct libraries rather than those that are incorrectly linked from the
+default location at `/opt/cray/pe/lib64` which is encoded into the executable's `RUNPATH`.
```
Comment (jsindt) [inline docs/known-issues/index.md:41]: Please could you restructure this sentence? It's not quite clear what you're trying to get across.

**Review by jsindt — COMMENTED** [review]
The final sentence of the FFTW section is a bit confusing -- please could you rephrase it? Other than that, it looks good to me.

File: docs/known-issues/index.md (hunk 1 above)
Comment (jsindt) [reply docs/known-issues/index.md:41]: Rereading, I think you can skip out the "at `/opt/cray/pe/lib64` which is encoded into the executable's `RUNPATH`" part.

File: docs/known-issues/index.md [hunk 2]
```diff
@@ -6,6 +6,40 @@ active investigation by HPE Cray and the wider service.
 
 ## Open Issues
 
+### OOM due to memory leak in libfabric (Added: 2022-02-23)
+
+There is an underlying memory leak in the version of libfabric on ArCHER2 (that comes as part of the underlying
```
Comment (juanfrh) [inline docs/known-issues/index.md:11]: Capitalisation of ARCHER2 is not correct.

File: docs/known-issues/index.md (hunk 1 above)
Comment (aturner-epcc) [reply docs/known-issues/index.md]: I have tried to clarify

File: docs/known-issues/index.md (hunk 2 above)
Comment (aturner-epcc) [reply docs/known-issues/index.md]: Fixed

File: docs/known-issues/index.md (hunk 1 above)
Comment (jsindt) [reply docs/known-issues/index.md]: Thanks -- it's very clear now. The "THe" at the start of the clarified section is not capitalised correctly.

**Comment by aturner-epcc** [comment]
@jsindt Good spot. Are you happy with this now?
